src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/PersistenceService.java

```diff
@@ -30,4 +34,15 @@ public void deleteRefreshJob(RefreshJobEntity refreshJobEntity) {
         refreshJobsRepository.delete(refreshJobEntity);
     }
 
+    public boolean getStatusByParam(String flagName, String envName) {
+        if (StringUtils.isEmpty(envName)) {
+            envName = System.getenv("LAUNCH_DARKLY_ENV");
```

Comment:
Please put "LAUNCH_DARKLY_ENV" in Constant file instead of hard code here.

Reply:
Done